composer.json

```diff
@@ -42,6 +44,9 @@
         "florianv/swap": "Exchange rates library for PHP",
         "psr/cache-implementation": "Used for Currency caching"
     },
+    "conflict": {
+        "florianv/swap": "<3.0"
```

Comment:
Haven't noticed it's in `require-dev`, so in fact it is possible to install it with lower Swap version, but as you said, it's optional dependency so there is no sense to restrict it.